Python/virtualenv.md

```diff
@@ -12,38 +12,50 @@ use `virtualenv` when dealing with Python applications.
 
 To install `virtualenv` run:
 
-    pip install virtualenv
+```sh
+pip install virtualenv
+```
 
 ## Usage
 
 If you have a project in a directory called `my-project` you can set up
 `virtualenv` for that project by running:
 
-    cd my-project/
-    virtualenv venv
+```console
+$ cd my-project/
+$ virtualenv venv
+```
 
 If you want your `virtualenv` to also inherit globally installed packages run:
 
-    virtualenv venv --system-site-packages
+```sh
+virtualenv venv --system-site-packages
+```
 
 These commands create a `venv/` directory in your project where all
 dependencies are installed. You need to **activate** it first though (in every
 terminal instance where you are working on your project):
 
-    source venv/bin/activate
+```console
+$ source venv/bin/activate
```

Comment:
Maybe I'm not understanding the logic behind some of these `$`. Could you explain why you added it? 🙂

Reply:
right, so i wanted to separate out content which is more of a post install command rather than the actual command needed for the user to use the application. That is the reason i went with the `$` to differentiate the actual instruction commands. Lemme know if you differ with this idea.

Reply:
It's easier to copy and paste without the `$`, and easier to read